cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class CinemaSerializer(serializers.Serializer):
```

Comment:
maybe u should call it MovieSerializer ?)

Reply:
yep))